ast/node.go

```diff
@@ -76,6 +83,8 @@ const (
 	NodeUnaryExpr
 	NodeMemberExpr
 	NodeCallExpr
+	NodeVarDecl
```

Comment:
```
if(var a = 1) { console.log("hello"); }
VM99:1 Uncaught SyntaxError: Unexpected token var
```
simple var decl is not permitted.

Reply:
I don't get the point

Reply:
https://github.com/NeowayLabs/abad/blob/a2703fa0a51873436a27124eb76cadb3cbced9ba/ast/node.go#L76
You defined the NodeVarDecl and NodeVarDecls beetwen `exprBegin` and `exprEnd`.
Take a look: https://github.com/NeowayLabs/abad/blob/a2703fa0a51873436a27124eb76cadb3cbced9ba/ast/node.go#L123

Just need to move it out of this section.